tools/hack/install.sh

```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/env bash
```

Comment:
Do we really need an install script?

Reply:
This is really needed, see https://github.com/kubernetes-sigs/ingress2gateway/pull/9#discussion_r1133004623 

Not everyone is familiar with go get/install, and krew. It is very important for UX. And this is how we recommend to install egctl now.

Reply:
Removed this script, let us add it if we needed later.